Modix.Services/Moderation/ModerationService.cs

```diff
@@ -227,7 +227,7 @@ public async Task UnConfigureGuildAsync(IGuild guild)
             AuthorizationService.RequireAuthenticatedUser();
             AuthorizationService.RequireClaims(AuthorizationClaim.DesignatedRoleMappingDelete);
 
-            using (var transaction = await DesignatedRoleMappingRepository.BeginDeleteTransactionAsync())
```

Comment:
This is actually a bug. `IRepositoryTransaction`s do not auto-commit. They auto-rollback if you haven't committed. There actually needs to be a call to `transaction.Commit();` at the end of this `using` block.

Reply:
I didn't know that

Reply:
If you want, you can submit that as a bugfix in a separate PR.

Reply:
Will do